aspnetcore/blazor/fundamentals/dependency-injection.md

```diff
@@ -213,7 +213,110 @@ The DI system is based on the DI system in ASP.NET Core. For more information, s
 
 ## Request a service in a component
 
-After services are added to the service collection, inject the services into the components using the [`@inject`](xref:mvc/views/razor#inject) Razor directive, which has two parameters:
+:::moniker range=">= aspnetcore-9.0"
+
+For injecting services into components, Blazor supports [constructor injection](#constructor-injection) and [property injection](#property-injection).
+
+### Constructor injection
+
+After services are added to the service collection, inject one or more services into components with constructor injection. The following example injects the `IDataAccess` service to obtain a list of actors for display by a component.
```

Comment:
This example is pretty elaborate considering it's really only trying to demonstrate ctor injection. It seems like this risks the user getting lost in all the other stuff that's going on, when the relevant bit is really just a single line of code. Thoughts?

Reply:
I agree now that you've mentioned it. This will require changes on the sample app side, too, so it's best for me to make the updates tomorrow morning.